packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.8.0
+
+* BREAKING CHANGE: Changed storage location for captured images and videos to internal cache on Android,
+to comply with new Google Play storage requirements.
```

Comment:
I think it would be good if we also explain here that if users want to keep the image, they are responsible to move it to a more persistent location. Maybe something like:

```suggestion
* BREAKING CHANGE: Changed storage location for captured images and videos to internal cache on Android,
to comply with new Google Play storage requirements. This means developers are responsible for moving the image or video to a more permanent location when it is required to  store the image or video permanently.
```

Reply:
Alright. I expanded a bit more on it as well, it should be a bit more clear now.